src/components/Accordion/AccordionItem.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Accordion.css';
+
+class AccordionItem extends React.Component {
+  constructor(props) {
+    super(props);
+    this.accordionItemToggle = this.accordionItemToggle.bind(this);
+    this.state = {
+      hidden: true,
+    };
+  }
+  accordionItemToggle() {
```

Comment:
Please add white space around functions.

Reply:
This is actually allowed so I've added whitespace.